### ISSUE TITLE
use the same settings on a new computer

### ISSUE THREAD

**Issue by theoneandonly-vector** [mRemoteNG/mRemoteNG#2136]
I would like to use the same settings I have on one computer on another computer

## Expected Behavior // ## Current Behavior

copying C:\Users\%username%\AppData\Roaming\mRemoteNG -> new PC same place
&
copying C:\Users\%username%\AppData\Local\mRemoteNG -> new PC same place

and then opening mRemoteNG should result in the same config on the new PC I thought, but it doesn't.

## Possible Solution
Tell me what else is needed :)

## Your Environment

* Version used: 1.76.20.24615 (on both pcs)
* OS: Windows10 21H1 19043.1466) (old)
* OS: Windows Server 2022 21H2 20348.202) (new)

**Comment by Kvarkas** [comment]
this is related to planed changes https://github.com/mRemoteNG/mRemoteNG/issues/2134
then you can save user preferences and after transfer or deploy just drop it into same folder with mRemoteNG and it will fetch user names and path to deploy/use it

**Comment by simonai1254** [comment]
@Kvarkas yes the user interface settings would make more sense when saved in the Appdata Roaming (as in the mentioned issue), though this does not explain why when copying all files the configuration is not used. 

@theoneandonly-vector The Appdata\Local\mRemoteNG folder includes another folder with random string (don't know where it's taken from or generated). I guess you might want to open mRemoteNG on the new machine and change a setting to generate the folder, and then put the _user.config_ in the generated folder.

**Comment by lars18th** [comment]
Hi,

I agree with the idea of this. But please consider a real PORTABLE mode. In this case, if all the configuration it's inside the same folder as the binaries (and not in the Appdata folder), then it will be used.

I hope you want to inplement in this way.
Thank you.

**Comment by Kvarkas** [comment]
As far as I know currently portable version keeps all its stuff in same folder, or is it not true?

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `a1717026` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** use the same settings on a new computer

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
